src/devices/Mpr121/Mpr121.cs

```diff
@@ -0,0 +1,252 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.ComponentModel;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Mpr121
+{
+    /// <summary>
+    /// Supports MPR121 Proximity Capacitive Touch Sensor Controller.
+    /// </summary>
+    public class Mpr121 : IDisposable, INotifyPropertyChanged
+    {
+        private static readonly int CHANNELS_NUMBER = Enum.GetValues(typeof(Channels)).Length;
+
+        private I2cDevice _device;
+        private Timer _timer = null;
+
+        private Dictionary<Channels, bool> _statuses;
+
+        private int _periodRefresh;
+
+        /// <summary>
+        /// Notifies about a property has been changed.
+        /// Refresh period can be changed by setting PeriodRefresh property.
+        /// </summary>
+        public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
Change to a more meaningful name? I think you can implement it with a custom event instead of using INotifyPropertyChanged. In my impression, INotifyPropertyChanged is commonly used in programs with interfaces. 😄

Reply:
Fixed.